deploy/cr.yaml

```diff
@@ -455,6 +455,7 @@ spec:
         cpu: 300m
   backup:
     image: perconalab/percona-xtradb-cluster-operator:main-pxc8.0-backup
+#    backupBackoffLimit: 6
```

Comment:
Can't we just name it `backoffLimit`? @hors @spron-in

Reply:
@egegunes up to @spron-in (in spec we have `backupBackoffLimit`)

Reply:
I'm ok with backoffLimit. Please suggest the change in the spec though.

Reply:
@spron-in couldn't find the spec for it, could you share the link?

@pooknull please also fix the example in CR

Reply:
fixed 61c49c6b8587b763d0583bcb550b2b15981598b1